.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,68 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: Description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: Tell us what you saw!
+    validations:
+      required: true
+  - type: textarea
+    id: how_to_reproduce
+    attributes:
+      label: How To Reproduce
+      description: Steps to reproduce the error.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '...'
+        3. Scroll down to '...'
+        4. See error"
+    validations:
+      required: true
+  - type: input
+    id: Expected_Behaviour
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen.
+      placeholder: "I expected the page to load the images."
+    validations:
+      required: true
+  - type: dropdown
+    id: device
+    attributes:
+      label: What device were you using when you encountered the error?
+      description: 
+      options:
+        - Desktop/Laptop
+        - Smartphone/Tablet
+    validations:
+      required: true
+  - type: dropdown
+    id: operating_system
+    attributes:
+      label: What operating system were you using?
+      options:
+        - Windows
+        - macOS
+        - Android
+        - iOS
```

Comment:
Need a version on this, just the name is not very helpful. If we update these to things like Windows 10, Windows 11, etc.